romana-install/romana-setup

```diff
@@ -49,14 +56,53 @@ if (( $# > 0 )); then
 				usage
 				exit 0
 				;;
-			-t|--target)
+			-n|--name)
+				case "$2" in
+					[a-zA-Z][a-zA-Z0-9]*)
+						stack_name="$2"
+						shift 2
+						;;
+					*)
+						echo "Invalid stack name: '$2' (must start with a letter, and contain only letters and digits)"
+						exit 1
+						;;
+				esac
+				;;
+			-p|--platform)
 				case "$2" in
-					aws|vagrant)
-						target="$2"
+					aws)
+						platform="aws"
+						required+=( "aws" )
+						shift 2
+						;;
+					vagrant)
+						platform="$2"
```

Comment:
Updated for the first suggestion. The 'platform' name is the public name of the platform, and 'required' is the CLI tool to provision things. If we added Google as a future platform, it might not have the same matching between those things.